### DISCUSSION TITLE
MountainDuck - SharePoint Integration Questions

### DISCUSSION THREAD

**Discussion by trulyvexed** [iterate-ch/cyberduck#16017] [Q&A]
Hoping someone will be able to answer these questions as I couldn't see them anywhere - apologies as there are a few of them!!

I'm trying to find a product that works on both Mac and Windows to access SharePoint as a mapped drive and most importantly is reliable.  

I couldn't find any information on how the SharePoint libraries are accessed on MountainDuck - Graph API? WebDav? 

If WebDav, I know other products on the market have issues with getting throttled on SharePoint so the drive becomes unresponsive, has anyone experienced this?

Does MountainDuck automatically enable the native Microsoft Office Collaboration tools (i.e Autosave, ability for multiple people to edit at once) when opened from SharePoint

Does the software have an autoupdate mechanism?

Does the Mac OS version require you to mess around with enabling Kernel Extensions?

If deployed via MSI / PKG, can we easily standardise on settings for all users (i.e force Online Only, prevent Sharing, configure which sites to connect t)

**Comment by dkocher** [comment]
- Mountain Duck uses the Microsoft Graph API to access SharePoint unless you explicitly select to to use _WebDAV_ instead [^1]. 
- Co-authoring for Microsoft Office documents is not supported.
- An update notification is shown when a newer version is available if selected in _Preferences → Update_
- No admin privileges or kernel extensions are required on macOS. Mountain Duck is also available in the Mac App Store running sandboxed.
- You can pre-configure settings by deploying a _default.properties_ file [^2] 

[^1]: https://docs.cyberduck.io/protocols/sharepoint/#sharepoint-server
[^2]: https://docs.mountainduck.io/mountainduck/preferences/#hidden-configuration-options
